### ISSUE TITLE
New version notify

### ISSUE THREAD

**Issue by andrewxyxyxy** [ExtendLord/DeezLoader-Reborn#107]
The notify of a new version is very good, but it could be really great with a link to the download page :)

**Comment by auanasgheps** [comment]
Yes, the new version notification is great... But where is _auto update_, mentioned in 3.0.3 changelog? Am I missing something?

**Comment by ExtendLord** [comment]
@auanasgheps Well the "auto update" is that nofitier.
Although its not actually updating, just notifying that there is an update.

**Comment by NicolasPL64** [comment]
There's no any way to update it automatically? Or is so hard/impossible to do?

**Comment by irule2day** [comment]
> There's no any way to update it automatically? Or is so hard/impossible to do?

All you need to do is click uninstall and install the new version. You literally do 2 steps. All your settings are saved separately. Is that so hard to do?

**Comment by andrewxyxyxy** [comment]
Ehi guys I was not talking about the installer program (that is really simple), I was only talking about adding a link to a download page that can be a little more difficult to find to someone that is not practice with the program. That's it :)
